scripts/linters/python_linter.py

```diff
@@ -117,6 +117,44 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_non_test_files(self):
+        """This function is used to check that function
+           with test_only in their names are in test files.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting function defintion checks')
+            python_utils.PRINT('----------------------------------------')
+        summary_messages = []
+        files_to_check = self.py_filepaths
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+            for filepath in files_to_check:
+                if not filepath.endswith('_test.py'):
+                    file_content = FILE_CACHE.read(filepath)
+                    for line_number, line in enumerate(
+                            file_content.split('\n')):
+                        ind1 = line.find(' def ')
```

Comment:
`startswith`*

Reply:
because of indentation, startswith will give false.
e.g . in python file
"    def test_only()"
this string starts with a balck space.

Reply:
Use `strip()`

Reply:
@Hudda I am just being curious here,sorry for trouble,
 yes i can change the code using strip, but how is the current code not right? 
as mentioned in my comment https://github.com/oppia/oppia/pull/9450#discussion_r436613826
I don't see how find is gonna create any problem, please give me a walkthrough so i can understand and follow for future PR reference.

Reply:
@Hudda okay, i will make the asked changes. Thanks.

Reply:
Consider the case `def list_test_only_elements()`, `find` will report this as True, but we want to check only for 'test_only' in the beginning of function's name.

Reply:
done